docs/ops/image/GridSample_1.md

```diff
@@ -0,0 +1,80 @@
+# GridSample {#openvino_docs_ops_image_GridSample_1}
+
+**Versioned name**: *GridSample-1*
+
+**Category**: *Image processing*
+
+**Short description**: *GridSample* performs interpolated sampling of pixels from the input image using normalized, non-integer coordinates passed in as one of its inputs.
+
+**Detailed description**: *GridSample* operates on a 4D input tensor representing an image. It calculates the output by selecting a location in the input image based on the values of the `grid` input. The latter contains a pair of float numbers for each output element that the operator is supposed to produce. Conceptually the operator behaves like *Gather* or *GatherND* but the difference is that the pixels to be selected are denoted by pairs of floats which belong to the range `[-1, 1]`. Those values have to be denormalized first (mapped to the integer coordinates of the input tensor) and then the output value is calculated according to the interpolation `mode`.
+
+**Attributes**
+
+* *align_corners*
+
+  * **Description**: controls how the extrema values in the `grid` input tensor map to the border pixels of the input image. The value of -1 for both width and height can map either to the center of the border pixels or their left/top border. Similarily the value of 1 can also map to the center of the border pixels or their right/bottom border. Inherently this means that the `GridSample` operation treats pixels as squares rather than infinitely small points.
+  * **Range of values**:
+    * `false` - map extrema values to the center of pixels
+    * `true` - map extrema values to the borders of pixels
+  * **Type**: `boolean`
+  * **Default value**: false
+  * **Required**: *no*
+
+* *mode*
+
+  * **Description**: specifies the interpolation type used to calculate the output elements
+  * **Range of values**: one of: `bilinear`, `bicubic` or `nearest`
+  * **Type**: `string`
+  * **Default value**: bilinear
+  * **Required**: *no*
+
+* *padding_mode*
+
+  * **Description**: controls the handling of out-of-bounds coordinates. The denormalized coordinates might fall outside of the input tensor's area(values outside the grid).
+  * **Range of values**: 
+    * `zeros` - consider values in the padding to be zeros
+    * `border` - the operator is supposed to select the nearest in-bounds pixel
+    * `reflection` - repeatedly reflect the out-of bounds value until it points to a pixel that belongs to the image
+  * **Type**: `string`
+  * **Default value**: zeros
+  * **Required**: *no*
+
+**Inputs**
+
+*   **1**: `data` - Input tensor with data to be sampled. Any supported OV data type can be used. This input is expected to be a 4-dimensional tensor with NCHW layout. **Required.**
```

Comment:
Please add *Type* section and use _T_ to describe types. You can take [Interpolate-4](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/image/Interpolate_4.md) spec as reference.